### PR TITLE
Add Note to Install Locally

### LINKED ISSUE #3045: No mention to not install Astro globally
Installing Astro globally is not recommended or supported, and often leads to weird issues that can be hard to debug. 

I think it would be judicious to add a section somewhere perhaps in the installation or getting started area to avoid this situation. 

Oftentimes, people install Astro globally by accident because they see a command being "astro dev" instead of "npm run dev", run that, see an error and Google tell them to install the package globally

### PR DESCRIPTION
**Pull request by Jothsa**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- Closes #3045 
- What does this PR change? I added a note to the getting started page and manual install guide warning users to not install Astro globally.

### CODE REVIEW

File: src/content/docs/en/install/manual.mdx [hunk 1]
```diff
@@ -59,6 +59,10 @@ Once you are in your new directory, create your project `package.json` file. Thi
 
 First, install the Astro project dependencies inside your project.
 
+:::note[Important]
```
Comment (sarah11918) [inline src/content/docs/en/install/manual.mdx:62]: I do like the note here. I think this is where people are installing manually, and are most likely to make this "mistake"!

**Review by sarah11918 — COMMENTED** [review]
This note looks good, @Jothsa! :raised_hands: Let's keep it to the manual installation page only, I think!

**Comment by Jothsa** [comment]
Thanks, @sarah11918! I think I accepted your changes correctly

**Comment by sarah11918** [comment]
Everything looks great to me! Once the tests pass, we're good to go!  :partying_face:

**Pull request merged**